src/App.css

```diff
@@ -1,38 +1,81 @@
-.App {
-  text-align: center;
+.bg {
+  background-image: url("https://user-images.githubusercontent.com/41516971/139524483-90a68abb-a68d-4cb3-bf56-5bde965c823c.jpg");
+  width: 100%;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 100vh;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.signUpBox {
+  border: 1px solid rgba(117, 117, 117, 0.5);
+  height: 65%;
+  text-align: -webkit-center;
+  padding: 35px 0 0 25px;
+    box-shadow: 0 3px 8px rgba(117, 117, 117, 0.5);
 }
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.signUpBtn {
+  width: 330px !important;
+  color: #5b79ff;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.google-btn {
+  margin: 15px 0 0 -20px;
+  width: 95%;
+  min-width: 184px;
+  max-width: 184px;
```

Comment:
just use `width: 184px;` no need to assign the same values in min and max width.